rosdep/base.yaml

```diff
@@ -4994,6 +4994,18 @@ r-base-dev:
   fedora: [R-devel]
   gentoo: [dev-lang/R]
   ubuntu: [r-base-dev]
+range-v3:
+  debian:
+    '*': [librange-v3-dev]
+    jessie: null
+    stretch: null
+  fedora: [range-v3]
+  gentoo: [dev-cpp/range-v3]
+  ubuntu:
+    '*': [librange-v3-dev]
+    artful: null
+    wily: null
```

Comment:
Same for artful and wily; since they are out-of-support, we don't need to add exceptions for them.

Reply:
Same here. Added those because they were in the list of supported platforms for Kinetic (wily) and Melodic (artful). Also removed these.